docs/content/glossary.md

```diff
@@ -323,7 +311,7 @@ The Receptionist service allows for a direct connection to the SpatialOS runtime
 
 > Also referred to as "synchronization".
 
-Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by either [standard](#standard-replication) or [custom](#custom-replication) replication.
+Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by either [standard](#standard-replication).
```

Comment:
```suggestion
Replication is the process by which all workers in the [SpatialOS world](#spatialos-world) maintain a consistent model of that world. It can be achieved by using the [standard replication](#standard-replication).
```

Reply:
Not sure if `the` is correct here

Reply:
Honestly I'd like to remove this entirely given that there is no standard replication or custom replication - just replication